Source/WTF/wtf/FixedVector.h

```diff
@@ -84,6 +84,12 @@ class FixedVector {
         : m_storage(size ? Storage::create(size).moveToUniquePtr() : nullptr)
     { }
 
+    FixedVector(size_t size, const T& value)
+        : m_storage(size ? Storage::create(size).moveToUniquePtr() : nullptr)
+    {
+        fill(value);
```

Comment:
I'm a little puzzled by this. The change log implies we are avoiding using `fill` so we can use more efficient constructors. But this constructor simply uses `fill`. Perhaps the main value in this change is avoiding initializing twice? Or are the constructors more efficient?

Or maybe the main problem is the caller who use `fill` to **resize** a vector. That’s definitely less efficient than sizing it correctly in the first place.

Reply:
Are there **any** good uses of `fill`?

Reply:
The perf benefits are for Vector only. The reason I added this constructor to FixedVector and BitVector is purely because one of the classes in JSC takes the vector type as template parameter and so I needed consistency in those vectors apis for that class to benefit from the more efficient Vector constructor. I think the consistency between those vector types is a plus too.

Reply:
And yes, they are plenty of remaining good uses of fill(), on vectors that were not just created. Often in functions that reset or update things.